### PR TITLE
Add usage to README

### PR DESCRIPTION
**Pull request by JBaczuk**
Give an example usage.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,15 @@
 # bcrpc
 Tiny Bitcoin RPC wrapper for node.js
+
+## Usage
+Each rpc function is exposed as a function an instantiated RpcAgent. The first argument is an array of arguments, expected by that specific rpc function, see [Bitcoin-core RPC Docs](https://bitcoincore.org/en/doc).
```
Comment (kallewoof) [inline README.md]: Typo: `as a function [in] an instantiated RpcAgent` (missing e.g. `in`)

**Comment by JBaczuk** [comment]
Fixed typo.

**Comment by JBaczuk** [comment]
Also, corrected erroneous statement about the first argument being an array and provided an example of error handling and a rpc function with a parameter.

**Comment by kallewoof** [comment]
Thanks!

**Pull request merged**